tests/api/test_api_feedback.py

```diff
@@ -25,6 +55,89 @@ def test_api_feedback(load_test_data, view_name, post_data):
     client = Client()
     url = reverse(view_name, kwargs={"region_slug": "augsburg", "language_slug": "de"})
     response = client.post(url, data=post_data)
-    print(response.headers)
     assert response.status_code == 201
     assert response.json() == {"success": "Feedback successfully submitted"}
+
+    # database checks
+    feedback = Feedback.objects.first()
+    assert feedback
```

Comment:
```suggestion
    assert Feedback.objects.count() == 1
    feedback = Feedback.objects.first()
```
You could also make sure that exactly one element is created and not multiple :sweat_smile: